File: app/src/main/java/com/walhalla/appextractor/adapter/viewholder/RecyclerViewSimpleTextViewHolder.java
package com.walhalla.appextractor.adapter.viewholder;

import android.view.View;
import android.widget.TextView;

import com.walhalla.appextractor.model.ViewModel;

public class RecyclerViewSimpleTextViewHolder extends BaseVh {

    private TextView text1;

    public RecyclerViewSimpleTextViewHolder(View view) {
        super(view);
        text1 = view.findViewById(android.R.id.text1);
    }

    @Override
    public void bind(ViewModel viewModel, int position) {
        text1.setText(viewModel.toString());
    }
}

File: app/src/main/java/com/walhalla/appextractor/adapter/viewholder/RecyclerViewSimpleTextViewHolder.kt
package com.walhalla.appextractor.adapter.viewholder

import android.R
import android.view.View
import android.widget.TextView
import com.walhalla.appextractor.model.ViewModel
import com.walhalla.appextractor.resources.StringItemViewModel


//class RecyclerViewSimpleTextViewHolder(view: View) : BaseVh<StringItemViewModel>(view) {
//    private val text1: TextView = view.findViewById(R.id.text1)
//
//    override fun bind(viewModel: StringItemViewModel, position: Int) {
//        text1.text = viewModel.text
//    }
//}


class RecyclerViewSimpleTextViewHolder(view: View) : BaseVh<ViewModel>(view) {
    private val text1: TextView =
        view.findViewById(R.id.text1)

    override fun bind(viewModel: ViewModel, position: Int) {
        text1.text = viewModel.toString()
    }
}